apis/feedapi/feedapi.yaml

```diff
@@ -0,0 +1,119 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: User Feed  API
+  description: >-
+    - The User Feed APIs  are used to store all alerts that could be presented to users. This includes tenant migration.
+host: staging.open-sunbird.org
+basePath: /api/
+securityDefinitions:
+  bearer:
+    type: apiKey  # arbitrary name for the security scheme
+    in: header # The secrity scheme parameter is in "header"
+    name: Authorization # name of the header, query parameter or cookie
+  userToken:
+    type: apiKey  # arbitrary name for the security scheme
+    in: header # The secrity scheme parameter is in "header"
+    name: x-authenticated-user-token # name of the header, query parameter or cookie
+schemes:
+  - https
+paths:
+  /user/v1/feed/{User_ID}:
+    get:
+      description: >-
+        This API gets all pending feeds for a user as identified by the User ID.
+
+      summary: Get user feed by User ID
+      tags:
+        - User Feed APIs
+      operationId: Get user by User ID
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: User_ID
+          type: string
+          required: true
+          description: 'Append a valid User Id To the Request URL.'
+        - name: ts
+          in: header
+          required: false
+          type: string
+          description: 'Timestamp at which given API request is sent.'
+        - name: X-msgid
+          in: header
+          required: false
```

Comment:
@anmol2302 if this is a unique identifier, is it optional?

Reply:
@Sunitashelar this is optional params in headers